app/api.py

```diff
@@ -462,6 +462,65 @@ def update_ticket_status(ticket_id: int, status: TicketStatus):
             raise HTTPException(status_code=404, detail="Not found")
 
 
+@api_v1_router.get("/get-data")
+def get_data(n_days: int = 31):
+    """Get number of tickets by status for the last n days.
+
+    Args:
+        n_days (int): The number of days from which to fetch ticket data.
+        Default is 7 days.
+
+    Returns:
+        dict: A dictionary containing the number of tickets for each status
+        (e.g., 'open', 'closed').
+    """
+    with db:
+        # Return the total number of tickets
+        total_tickets = TicketModel.select().count()
+
+        # Get the total number of tickets created in the last n days
+        start_date = datetime.utcnow() - timedelta(days=n_days)
```

Comment:
`datetime.utcnow is deprecated`, due to the way naive datetimes are represented by the `datetime` module (https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow). You should use `datetime.now` (https://docs.python.org/3/library/datetime.html#datetime.datetime.now) providing the UTC as parameter (`datetime.now(timezone.utc)`)